sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/SignalingConfig.java

```diff
@@ -0,0 +1,15 @@
+package com.azure.android.communication.chat.signaling;
+
+public class SignalingConfig {
```

Comment:
Is this a type we want to expose to the end user? if not we can make it package private or move to implementation package.